EndlessSky.cbp

```diff
@@ -320,6 +320,8 @@
 		<Unit filename="source/TradingPanel.h" />
 		<Unit filename="source/UI.cpp" />
 		<Unit filename="source/UI.h" />
+		<Unit filename="source/Variant.cpp" />
+		<Unit filename="source/Variant.h" />
```

Comment:
>(I don't have a Mac, so I cannot give you the exact change that is needed.)

Well I don't have a Mac either, so we'll need someone with the knowledge of updating that to do it.

Reply:
This would be blocking before merging this change. You could ask somebody on Discord that has a Mac to provide you the contents. I also did that for formation flying and got a response on the required change.

Reply:
I was asking around but got no responses.

Reply:
Sounds like we need something more structurally for this. I have the same issue with the MacOS change for #4370. Maybe we can get some CI-job to automatically generate this for us.

Reply:
I now have a structural solution, #4817 implements a CI-job that automatically generates the diff to apply to EndlessSky.xcodeproj/project.pbxproj . I used the script in the CI-job to generate the Xcode changes for #4370 (and that seems to build fine).